### DISCUSSION TITLE
Real V Maker?

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#18878] [General]
Well, according to [this](https://dotcommagazine.com/2023/07/vlang-top-five-important-things-you-need-to-know/), a guy named Danilo created V.  So when is team V going to introduce us to him?

**Comment by Delta456** [comment]
@medvednikov is the creator of V, not him.

**Reply by Wajinn** [reply]
Totally.  But maybe team V needs to tell them that.  Might cause some confusion.

**Reply by JalonSolov** [reply]
Tried to send them an email, but their email button opens the email with the `To:` field blank.

**Reply by Wajinn** [reply]
Interestingly, they are blue checked and on [Twitter](https://www.twitter.com/magazinedotcom), to include on [YouTube](https://www.youtube.com/@dotcommagazine).  Also, the site is owned and operated by Tech Team LLC.  Though that gets a bit fuzzy of which Tech Team LLC is being referred to.  There are lots of companies and sites which use variations of this.

**Comment by medvednikov** [comment]
This article feels like it's generated by chatgpt :)

**Reply by JalonSolov** [reply]
I think that particular name of the "creator" of V has come up in chatgpt before too.

**Reply by Wajinn** [reply]
It's kind of fascinating, about how that name pops up on chatgpt and how someone would use/abuse it like that.  I understand having "writer's block" and to use it to get something started, but the person should be in control of it, not the other way around.
